models/sales/sales.yml

```diff
@@ -18,8 +18,6 @@ models:
       - name: deal_id
         description: The ID of the deal
         tests:
-          - dbt_expectations.expect_column_values_to_be_unique:
-              row_condition: "not coalesce(is_deal_deleted, false)"
           - unique
```

Comment:
Shouldn't this be a not_null test as well?

Reply:
What first test are you referring to?

Reply:
Same for other comments

Reply:
@fivetran-joemarkiewicz sorry to miss your response-- I'm referring to line 23, just noticing how uniqueness was being tested twice for the `deal_id` field, both via the generic dbt test and the dbt_expectations macro. I figured this was meant to be a `not_null` test instead. See [company_id](https://github.com/fivetran/dbt_hubspot/blob/da2ebaff67b7100127cb7228568547965441b447/models/sales/sales.yml#L131-L133) in hubspot__companies